x-pack/plugins/saved_objects_tagging/public/management/components/_action_bar.scss

```diff
@@ -0,0 +1,11 @@
+.tagMgt__actionBar {
+  > .euiHorizontalRule {
+    margin: 8px 0 3px;
+  }
+}
+
+.tagMgt__actionBar + .euiSpacer {
+  display: none;
+}
```

Comment:
https://github.com/elastic/eui/pull/4103 unblocked us by allowing to inject arbitrary content between the `InMemoryTable`'s searchbar and table. However the `EuiSpace` added when using `childrenBetween` conflicts with the expected design, so I have to hack it into `display: none;`

Reply:
Unfortunately, this whole file goes against our policies for custom styles on top of EUI. There are a couple of issues:

1. Don't target `.eui` classes. All our component accept a `className` prop which you can use to create your own class instance and append to the EUI component class list. Then target this class name so that, in case EUI ever changes it's naming structure (most likely to happen within the next year), your styles won't break.
2. Don't use hard-coded pixel values. Always use our [SASS variables](https://elastic.github.io/eui/#/guidelines/sass) for sizing and font sizes. Most likely a prop exists on the component to help this as well.

If you're having trouble affecting change on an EUI component, please let us know before targeting the `.eui` classes directly. Most likely we can get a fix in pretty quickly, or if there's a long-term solution being worked on, for instances like these, it's better to be consistent with the EUI components than to override them.

Reply:
> Don't target .eui classes. All our component accept a className

Unfortunately I can't here. I'm targeting the `EuiSpacer` that is inside the `EuiInMemoryTable`. See https://github.com/elastic/eui/pull/4103.

Reply:
Yep. Mostly my comment for that specifically is to say, please ask us or create a PR on the EUI side so we can support your needs without needing hacks. I see this as a valid request that we shouldn't really be introducing a hard-coded spacer here and leave it up to the consumer to provide one.

Reply:
Created https://github.com/elastic/eui/issues/4246

Reply:
Thanks @pgayvallet ! I've already got a PR up for you to fix this https://github.com/elastic/eui/issues/4246

I'll let it slide for now 😉 because at least when the fix merges into Kibana, your hack won't have any ill effect. I'd def then appreciate a follow-up.

Reply:
Created https://github.com/elastic/kibana/issues/83161. Will do it as soon as https://github.com/elastic/eui/issues/4246 get shipped in next Kibana EUI update. 

Thanks again for the quick fix btw